docs/manual/troubleshooting.tex

```diff
@@ -823,7 +837,9 @@ \subsection{Build the Type Annotations compiler\label{building-compiler}}
 (The \<JAVA\_HOME> environment
 variable might already be set, because it is needed for Ant to work.)
 
-In the bash shell, putting the following command in your \code{.bashrc} file
+\begin{enumerate}
```

Comment:
Write brief introductory text saying to choose one of the following alternatives.